configs/sst_cs_plumbers-unwanted.yaml

```diff
@@ -24,6 +24,7 @@ data:
   - libtar
   - libtar-devel
   - network-scripts
+  - numad
```

Comment:
Do you want to mark pinfo as explicitly unwanted here as well?

Reply:
Addressed in 5fc82686e6b4e1d0f001a03136c80315a319badf.